src/ypd/model/user.py

```diff
@@ -1,13 +1,14 @@
-from sqlalchemy import Column, Integer, String, ForeignKey, Boolean
+from sqlalchemy import Boolean, Column, ForeignKey, Integer, String
 
 from . import Base, Session
 
+
 class User(Base):
     """A class that represents a single user account"""
     __tablename__ = 'users'
     id = Column(Integer, primary_key=True)
-    username = Column(String)
-    password = Column(String)
+    username = Column(String, unique=True, nullable=False)
+    password = Column(String, unique=True, nullable=False)
```

Comment:
Passwords shouldn't need to unique.
It's fine for now, but we'll have to fix it as soon as we finish the demo

Reply:
Actually, you'll have to fix the other comment, so may as well fix this too